pyo3-ffi/build.rs

```diff
@@ -92,7 +89,7 @@ fn configure_pyo3() -> Result<()> {
     ensure_python_version(&interpreter_config)?;
     ensure_target_pointer_width(&interpreter_config)?;
 
-    if !interpreter_config.suppress_build_script_link_lines {
+    if is_linking_libpython() && !interpreter_config.suppress_build_script_link_lines {
```

Comment:
This seems to have the effect that `is_linking_libpython()` now also controls whether ` interpreter_config.to_cargo_dep_env()?;` is called or not. Is that intentional? Shouldn't we serialize the configuration into those variables in any case?

Reply:
Good catch! It's the result of rebasing on top of https://github.com/PyO3/pyo3/pull/2092 and an automatic git merge.

I'll fix this in the new revision.

Reply:
Fixed in https://github.com/PyO3/pyo3/commit/9b45a19161cd20fac1c83c25897aa9d213244a15